### ISSUE TITLE
Function to void unsegmented media intervals

### ISSUE THREAD

**Issue by yrose** [phon-ca/phon#183]
This function would involve actually editing audio files, as follows:

For each time interval *not* segmented within a Phon session, replace the audio content of the interval by nothing (no sound) or by some low-intensity pink noise 

This function could also include some buffer, such that 1-2 seconds of audio is preserved around the segment times. For example, using  a 2-record transcript and a 2-second buffer:

Record 1: 30-35 seconds
Record 2: 45-50 seconds

Phon would void the audio :
 - from the beginning of the file to 28 seconds
 - between 37 and 43 seconds
 - from 52 second until the end of the file

**Comment by ghedlund** [comment]
I can see two use cases for this operation. In either case the user should have the option of creating a new session or update current records with new segment times. A record selection form should also be included.

The two use cases are:

1) 'Minimize' audio
This action would reduce the current audio file to only audio samples relevant to records with configurable dead space between records. It would be difficult to update TextGrid information with this operation as Phon has no concept of the TextGrid classes which are controlled by the plug-in.

2) Void audio (for anon purposes possibly)
This action would 'Preserve audio file times' (this is how the option will be presented to the user - along with a warning about TextGrid data if not selected.) The audio file would be generated containing only record samples selected by user with everything else blank.

**Comment by yrose** [comment]
I agree with both options, and understand the issue about the first one. We could also decide to go for (2) only if this makes things easier. The length of the media files isn't normally a concern, so 'minimize' is likely less a priority than 'void'

In both cases, there should be minimal buffer zones for the intervals that are voided/removed, as per my original description.

**Comment by ghedlund** [comment]
Current recommendation for the name of the operation is 'Reconstruct Audio'
